desktop/core/src/desktop/lib/botserver/api.py

```diff
@@ -43,3 +44,20 @@ def get_channels(request):
   return JsonResponse({
     'channels': bot_channels,
   })
+
+@api_error_handler
+def send_message(request):
+  channel = request.POST.get('channel')
+  message = request.POST.get('message')
+
+  slack_response = _send_message(channel, message)
```

Comment:
How about adding the name of the user? (as it seems that the author is shown-up is the SQL Bot, which is probably confusing)

Reply:
(especially that we don't check if a user has perms to post to the Slack channel, is there a way/API to validate that?)

Also, when listing channels, I guess we list all the public ones as the hue bot user, but that's fine for now.

Reply:
:+1: